receiver/filereceiver/receiver.go

```diff
@@ -0,0 +1,31 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package filereceiver // import "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/filereceiver"
+
+import (
+	"context"
+
+	"go.opentelemetry.io/collector/component"
+)
+
+type fileReceiver struct{}
```

Comment:
If you'd like the reduce the lines of code you need to write, you can rerwrite this as:

```suggestion 
type fileReceiver struct {
    component.StartFunc
    component.ShutdownFunc
}
```

Reply:
Interesting, I wasn't familiar with that. Updated.